src/background/services/balances/handlers/getTotalBalanceForWallet/helpers/getIncludedNetworks.ts

```diff
@@ -0,0 +1,28 @@
+import { uniq } from 'lodash';
+import { ChainId } from '@avalabs/core-chains-sdk';
+
+import { ChainListWithCaipIds } from '@src/background/services/network/models';
+
+export function getIncludedNetworks(
```

Comment:
https://github.com/user-attachments/assets/7ca76ead-8ec9-405a-bf81-9c619790a792

Something is off when the user is on P-Chain and is switching between P and C. I have P chain favorited

Reply:
I think it's because C-Chain is not _actually_ a favorite network - we just always show it. Let me try to fix that.

Reply:
Yeah, so that was actually the case (fixed in 634ca1643).

Since C-Chain is not really a favorite network, it's not included in the total balance if it's not the active network. See the prod build on Bitcoin, for example:

![image](https://github.com/user-attachments/assets/bdf1ed30-92f9-4e28-b923-2ecb082e5564)

versus with the fix (C-Chain balance included):

![image](https://github.com/user-attachments/assets/d67226ad-cedb-461e-a01e-e96d513a6376)